plumbing/object/change_adaptor_test.go

```diff
@@ -1,11 +1,10 @@
-package difftree
+package object
```

Comment:
Since you rename the change_adaptor.go file to difftree.go, you should include this tests to difftree_test.go, a and rename the suite, otherwise nobody will know what change_adaptor is.